pkg/logs/client/http/sync_destination.go

```diff
@@ -0,0 +1,108 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//nolint:revive // TODO(AML) Fix revive linter
+package http
+
+import (
+	"sync"
+	"time"
+
+	"github.com/DataDog/datadog-agent/comp/logs/agent/config"
+	pkgconfigmodel "github.com/DataDog/datadog-agent/pkg/config/model"
+	"github.com/DataDog/datadog-agent/pkg/logs/client"
+	"github.com/DataDog/datadog-agent/pkg/logs/message"
+	"github.com/DataDog/datadog-agent/pkg/logs/metrics"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// SyncDestination sends a payload over HTTP and does not retry.
```

Comment:
Do we want to retry eventually? To remediate issues from 19254

Reply:
We used to retry here, but that just created duplicate logs. Alessandro moved the retry logic into our own serverless logs/trace/metrics flushing package and stopped retrying at the logs destination.

Reply:
Sounds good, thanks for clarifying, maybe add that in a comment here? That since this will only be used by Serverless, then we don't need to retry here?